packages/shopify-api/lib/webhooks/validate.ts

```diff
@@ -51,7 +51,10 @@ export function validateFactory(config: ConfigInterface) {
       await log.debug('Webhook request is valid', loggingContext);
       return webhookCheck;
     } else {
-      await log.debug('Webhook validation failed', loggingContext);
+      await log.debug('Webhook HMAC validation failed.', loggingContext);
+      await log.debug(
```

Comment:
Yeah I think it'll come down to personal preference - I usually prefer a single entry because it's easier to read through in non-console environments.